src/components/header/header.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import classes from './Header.module.scss';
+
+function getTitle() {
+  let title = 'Error 404';
+  const url = location.pathname;
+  if (url === '/') {
```

Comment:
space around if block, also u can create a folder utils and hide this function there, and use function expression here with arrow function